x/checkpoint/types/msg.go

```diff
@@ -132,8 +132,9 @@ func (msg MsgCheckpointAck) Route() string {
 
 // GetSigners returns signers
 func (msg MsgCheckpointAck) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{sdk.AccAddress([]byte(msg.From))}
-
+	//return []sdk.AccAddress{sdk.AccAddress([]byte(msg.From))}
+	addr, _ := sdk.AccAddressFromHex(msg.From)
```

Comment:
remove commented line if not required. Also, add an error handler when typecasting AccAddress.